src/docs/tutorials/proximity-steering.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Setting up Proximity Steering using Cloudflare
+---
+
+### What is Proximity Steering?
+
+>Proximity steering routes visitors ... to the closest physical data center.
+
+*Source: [Proximity steering](https://developers.cloudflare.com/load-balancing/understand-basics/traffic-steering/steering-policies/proximity-steering/)*
+
+## About this Tutorial
+
+As Railway does not offer native Proximity Steering at this time, we instead need to place Cloudflare in front of our services to do this for us.
+
+This tutorial aims to provide a simple step-by-step guide on setting everything up on Cloudflare to ensure Proximity Steering works flawlessly!
+
+**Objectives**
+
+In this tutorial, you will learn how to -
```

Comment:
I would clarify that each of these objectives refer to actions that will be performed in CloudFlare.  e.g.. "... you will learn how to do the following in CloudFlare - "